src/library_fs.js

```diff
@@ -701,7 +699,7 @@ FS.staticInit();
     mkdev(path, mode, dev) {
       if (typeof dev == 'undefined') {
         dev = mode;
-        mode = 438 /* 0666 */;
+        mode = {{{ 0666 }}};
```

Comment:
Can this not be just `0666` without the `{{{ }}}` ?

Reply:
Done